modules/appeals_api/app/controllers/concerns/appeals_api/openid_auth.rb

```diff
@@ -46,10 +46,6 @@ def token_validation_client
     end
 
     def validate_auth_token!
-      # Token validation can be skipped during local development by setting
-      # modules_appeals_api.enable_unsafe_mode = true in settings.local.yml
-      return if unsafe_mode?
-
```

Comment:
Oh, I'm sorry - thanks for asking - I forgot to explain this:

Yes, Bruno makes the removal less burdensome, but primarily: once we start validating the ICN that's provided in a token, this would actually stop the API from working correctly when in "unsafe mode", because we actually expect accurate information from the token going forward (we'd constantly get failed requests because we wouldn't be able to get an ICN from a token to validate)

Reply:
Oh, perfect! That makes sense. Thanks for walking through it for me!